examples/tutorials/hello-gltf/app.ts

```diff
@@ -27,11 +27,28 @@ export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
       throw new Error('This demo is only implemented for WebGL2');
     }
 
-    this.loadGLTF('Avocado');
+    window.localStorage.lastGltfModel = window.localStorage.lastGltfModel ?? 'Avocado';
+    this.loadGLTF(window.localStorage.lastGltfModel);
+
+    const modelSelector = document.getElementById('model-select') as HTMLSelectElement;
+
+    fetch('https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Assets/main/Models/model-index.json')
```

Comment:
Nits: 
- use async/await
- break out helper functions fetchModelList(), updateModelSelector() to keep the main code as clean and simple.

This app is in the tutorials section, so it should be kept clean and small.